### DISCUSSION TITLE
Font Awesome 6 for Joomla 4

### DISCUSSION THREAD

**Discussion by 4digitalworld** [joomla/joomla-cms#32363] [General] [answered]
### Is your feature request related to a problem? Please describe.
I suggest using [Font Awesome 6](https://github.com/FortAwesome/Font-Awesome) in @Joomla 4.

### Describe the solution you'd like
Replace Font Awesome 5.15.2 with Font Awesome 6

### Additional context
Currently Font Awesome 6 is [used](https://fontawesome.com/how-to-use/on-the-web/using-with/wordpress) in @WordPress.

I believe this is the right step for the development of Joomla, as developers will be targeting a specific version of the font in the future. Stable version Font Awesome 6 will be released this year, so Font Awesome 6 will develop along with Joomla 4 and Bootstrap 5. I think this is the right step. 

V6: [https://fontawesome.com/6](https://fontawesome.com/6)
V6 updates: [https://fontawesome.com/updates](https://fontawesome.com/updates)
Version: 6.0.0-alpha2: [https://fontawesome.com/download](https://fontawesome.com/download)

**Comment by Stuartemk** [comment]
I agree, you have to avoid a waste of time as happened when waiting so long to switch to Bootstrap 5.

**Comment by dgrammatiko** [comment]
Font Awesome needs to publish the free version 6 on npm for us to be able to consume it. Right now this is not the case, latest version 5.15.2
![Screenshot 2021-02-10 at 19 24 42](https://user-images.githubusercontent.com/3889375/107554154-db840300-6bd5-11eb-8650-bbec6db13558.png)

**Comment by 4digitalworld** [comment]
@tagliala @davegandy @robmadole @supercodepoet @talbs can you help us? Font Awesome 6 can be added to the brand new version of @Joomla 4 coming this spring! @fortawesome

**Reply by tagliala** [reply]
Hi, how we can help?

FA 6 is still in private alpha, I do not have an ETA for the stable release

**Comment by richard67** [comment]
According to their release schedule there will not be any alpha or beta version available on NPM before Winter, they will have only downloads for testing:

https://blog.fontawesome.com/version-6-release-schedule/

@sanek4life You could investigate such things yourself before proposing things here, then you would maybe see that your proposal does not make any sense.

**Reply by 4digitalworld** [reply]
I saw that there is already an official plugin for WordPess. Could such a plugin appear for Joomla? 

https://fontawesome.com/6#more-services

**Comment by 1337180** [comment]
https://fontawesome.com/6#more-services
Will J4 be in the official list of plugins?

**Comment by N6REJ** [comment]
FA has been part of joomla for years.  when 6 is ready I'm sure we'll update to it but not till then.

**Comment by N6REJ** [comment]
NPM for 6 is now released https://fontawesome.com/docs/web/setup/packages

https://www.npmjs.com/package/@fortawesome/fontawesome-free

**Comment by N6REJ** [comment]
This is going to require a major rewrite of the cross reference table(s) as there are major b/c changes in icon names.
https://fontawesome.com/docs/web/setup/upgrade/whats-changed#full-style-names

**Reply by brianteeman** [reply]
> But, again, don't worry if you have a project using Version 5. We made aliases for the old names that will translate to the new names if you don't want to update your references.

**Comment by dgrammatiko** [comment]
There’s a pr already https://github.com/joomla/joomla-cms/pull/36906

**Comment by joomleb** [comment]
Hi guys,
What is the updated FA6 roadmap ?

**Reply by brianteeman** [reply, marked as answer]
fontawesome 6.4 will be available in Joomla 5 - released in a few months

**Reply by joomleb** [reply]
@brianteeman 
Thank you very much for the info update
